ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/tasks/DownloadHeaderSequenceTask.java

```diff
@@ -163,6 +163,7 @@ protected boolean isRetryableError(final Throwable error) {
           BlockHeader child = null;
           boolean firstSkipped = false;
           for (final BlockHeader header : headersResult.getResult()) {
+            resetRetryCounter();
```

Comment:
Sorry I missed this - we only want to reset the counter after we get through the first header.  See `firstSkipped` below.